tests/unit/test_charm.py

```diff
@@ -91,6 +91,28 @@ def setup_loki_relation(harness: Harness) -> int:
     )
 
 
+def setup_tracing_relation(harness: Harness) -> int:
```

Comment:
It seems that this method returns nothing. Probably it's expected to return the `relation_id`.